src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -548,7 +591,7 @@ public TimeSpan GetTimeoutOrDefault(TimeSpan? timeout)
             SupervisorStrategy supervisorStrategy,
             CancellationToken cancellationToken = default)
             => ChildActorOfAsync(props, name, supervisorStrategy, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use `GetAwaiter().GetResult()` because this automatically unwraps all `AggregateException`, using `.Result` here breaks behavior consistency.